requirements.txt

```diff
@@ -40,6 +40,7 @@ pyparsing>=3.1.1,<3.2
 python-dateutil>=2.8.1
 python-redis-lock[django]>=4,<4.1
 rapidfuzz>=2.6.0,<3.5
+regex>=1.0
```

Comment:
There is no such version, please choose something reasonably recent as lower bound. Adding upper bound is also a good idea to avoid accidental breakage.